kernel/packages/unity-interface/BrowserInterface.ts

```diff
@@ -552,6 +559,20 @@ export class BrowserInterface {
     await killPortableExperienceScene(data.portableExperienceId)
   }
 
+  public RequestBIWCatalogHeader() {
+    const store: Store<RootState> = globalThis['globalStore']
+    const identity = getCurrentIdentity(store.getState())
+    if (!identity) {
+      let emptyHeader: Record<string, string> = {}
+      unityInterface.SendBuilderCatalogHeaders(emptyHeader)
+    } else {
+      const headers = BuilderServerAPIManager.authorize(identity, 'get', '/assetpacks')
+      headers['Content-Type'] = 'application/json'
```

Comment:
What is this for? Content-Type header is not valid for GET requests

Reply:
It was a copy & paste fail, thanks for the heads up! Fixed!